pytest_watch/watcher.py

```diff
@@ -39,7 +39,7 @@ def watch(directory=None, auto_clear=False, extensions=[]):
     """Starts a server to render the specified file or directory containing a README."""
     if directory and not os.path.isdir(directory):
         raise ValueError('Directory not found: ' + directory)
-    directory = os.path.abspath(directory)
```

Comment:
Does this actually fail?

When I run `os.path.absapth(None)` in the interpreter, it returns the absolute path of the current working directory. (On Windows anyway.)